src/components/WorkPoolEditForm.vue

```diff
@@ -32,24 +41,40 @@
 <script lang="ts" setup>
   import { showToast } from '@prefecthq/prefect-design'
   import { useValidationObserver } from '@prefecthq/vue-compositions'
-  import { ref } from 'vue'
+  import { ref, computed } from 'vue'
   import { useRouter } from 'vue-router'
-  import { SubmitButton } from '@/components'
-  import { useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
+  import { SubmitButton, WorkPoolTypeSelect, SchemaFormFieldsWithValues, SchemaPropertiesKeyValues } from '@/components'
+  import { useCan, useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
   import { localization } from '@/localization'
-  import { WorkPool } from '@/models'
+  import { WorkPool, WorkPoolEdit } from '@/models'
+  import { mapper } from '@/services/Mapper'
+  import { Schema } from '@/types/schemas'
+  import { getSchemaDefaults } from '@/utilities/parameters'
 
   const props = defineProps<{
     workPool: WorkPool,
   }>()
 
   const api = useWorkspaceApi()
+  const can = useCan()
   const router = useRouter()
   const routes = useWorkspaceRoutes()
   const { validate, pending } = useValidationObserver()
 
   const description = ref<string | null | undefined>(props.workPool.description)
+  const type = ref<string>(props.workPool.type)
   const concurrencyLimit = ref<number | null | undefined>(props.workPool.concurrencyLimit)
+  const schema = computed<Schema>(() => mapper.map('SchemaResponse', props.workPool.baseJobTemplate.variables ?? {}, 'Schema'))
+  // Set parameters to the default values from the schema so they are pre-filled in the form
+  const parameters = ref(
+    getSchemaDefaults(props.workPool.baseJobTemplate.variables ?? {}),
+  )
```

Comment:
This should already be happening in the `SchemaFormFieldsWithValues` component in its values getter. Is this necessary to do it again here?

Reply:
The default values were rendered as placeholders instead of populating the inputs with the default values. Is populating the inputs with the default values something that is already built into `SchemaFormFieldsWithValues`?

Reply:
It should be based on what I'm seeing https://github.com/PrefectHQ/prefect-ui-library/blob/edit-work-pool-base-job-template/src/components/SchemaFormFieldsWithValues.vue#L25

Reply:
After some inspection, the function that's called there gets default values based on the type (empty string for a `string` type, `null` for a `boolean` type). I'm getting the default value defined in the schema and using that for the values in the inputs, which is why this `getSchemaDefaults` call is necessary.

Reply:
Oh I see, I confused the two utilities. Thanks for clarifying